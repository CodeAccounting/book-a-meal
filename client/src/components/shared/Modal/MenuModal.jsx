import React from 'react';
import MenuForm from '../Form/MenuForm';

/**
 * @exports
 * @function MenuModal
 * @returns {component} MenuModal
 */
const MenuModal = () => <MenuForm />;

export default MenuModal;
