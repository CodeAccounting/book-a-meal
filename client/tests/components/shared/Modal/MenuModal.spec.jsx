import React from 'react';
import MenuModal from '../../../../src/components/shared/Modal/MenuModal';

describe('MenuModal', () => {
  afterAll(() => {
    jest.clearAllMocks();
  });

  it('renders correctly', () => {
    const shallowWrapper = shallow(<MenuModal />);

    expect(toJson(shallowWrapper)).toMatchSnapshot();
  });
});
