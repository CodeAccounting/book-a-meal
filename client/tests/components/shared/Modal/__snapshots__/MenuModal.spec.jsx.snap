// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuModal renders correctly 1`] = `<Connect(MenuForm) />`;
